.github/workflows/ci.yml

```
@@ -77,6 +77,54 @@ jobs:
+        shell: bash
+        run: |
+          XML=build/reports/jacoco/test/jacocoTestReport.xml
+          if [[ -f "$XML" ]]; then
+            line_missed=$(grep -o "counter type=\"LINE\"[^>]*" "$XML" | sed -E 's/.*missed=\"([0-9]+)\".*/\1/')
+            line_covered=$(grep -o "counter type=\"LINE\"[^>]*" "$XML" | sed -E 's/.*covered=\"([0-9]+)\".*/\1/')
+            branch_missed=$(grep -o "counter type=\"BRANCH\"[^>]*" "$XML" | sed -E 's/.*missed=\"([0-9]+)\".*/\1/')
+            branch_covered=$(grep -o "counter type=\"BRANCH\"[^>]*" "$XML" | sed -E 's/.*covered=\"([0-9]+)\".*/\1/')
+            line_total=$((line_missed + line_covered))
+            branch_total=$((branch_missed + branch_covered))
+            line_pct=$(python3 - <<PY
+from decimal import Decimal, ROUND_HALF_UP
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 94: could not find expected ':'

(syntax-check)